### DISCUSSION TITLE
A 'Collapse All Groups' option in two menus...

### DISCUSSION THREAD

**Discussion by JonEBee** [nextgenhealthcare/connect#5489] [Feature/Enhancement Ideas]
1. In the Dashboard Tasks menu in the Dashboard page, and
2. In the Group Tasks menu in the Channels page.

**Comment by pacmano1** [comment]
you can already to that in the upper right corner.  little square on upper right.

**Reply by pacmano1** [reply]
<img width="77" alt="image" src="https://user-images.githubusercontent.com/44065187/197344648-296c6ed6-c9ff-42f4-9e5b-82e993d5fb8b.png">
